server/util/invocationutil/invocationutil.go

```diff
@@ -0,0 +1,34 @@
+package invocationutil
```

Comment:
Awesome that you factored this out!

Could we call it something different than util/invocationutil tho? It stutters (util), and also i don't think it's broadly useful.

How about we put a package called "format" or "invocation_format" inside of server/build_event_protocol?

Reply:
sg, I wasn't happy with the name invocationutil either :P moved to `server/build_event_protocol/invocation_format`